tests/unit/test_tf_estimator.py

```diff
@@ -305,6 +306,68 @@ def test_create_model_with_optional_params(sagemaker_session):
     assert model.vpc_config == vpc_config
 
 
+@patch("sagemaker.tensorflow.serving.Model.transformer")
```

Comment:
sorry, I think I explained this incorrectly last time...
* You should still mock `create_model` because you want to assert if that is called with `endpoint_type` set correctly, and you don't actually care about what that method is doing (since presumably there will be unit tests for that method).
* You also want to make sure `transformer()` isn't called for real, since again, it's beyond the scope of this unit test.
* Lastly, you do want to still check that `transformer()` was called with the correct args, since that's part of the logic of the method you wrote.

so keep the patch on `create_model`, and make sure it's `return_value` is a `Mock` object that you can reference later:

```python
# create_model is the patched method
model = Mock()
create_model.return_value = model

# your existing test code

create_model.assert_called_with(...)
model.transformer.assert_called_with(...)
```

Reply:
fixed